tests/test_api.py

```diff
@@ -2083,6 +2083,48 @@ def test_delegate(self) -> None:
             "subscription_id": f"0/{expected_uuid.hex}",
         }
 
+    def test_delegate_with_sessions_entity_subscription(self) -> None:
```

Comment:
I think `test_api.py` is generally for testing events, there is `test_sessions_api.py` for sessions API tests.